app/javascript/graph_builder.js

```diff
@@ -6,6 +6,7 @@ import {build_reset_graph} from "graphs/reset_graph.js"
 import {chartOptions} from "consts.js"
 import {Spinner} from 'spin.js'
 import Highcharts from 'highcharts'
+import _ from 'lodash'
```

Comment:
Where is the lodash library coming from? I don't see any occurrence of "lodash" in our code.

Reply:
So it turns out lodash and underscore are basically the same thing, with lodash being spawned off from underscore in the past.  I saw the references to `_.xxx` functions so I messed up thinking we had it, and because webpack-merge pulls it in it wasn't causing an error.  I corrected this and found an error in the upload file that I also fixed when removing underscore from the global namespace.